docs/api.md

```diff
@@ -1724,6 +1747,18 @@ puppeteer.launch().then(async browser => {
 
 In the case of multiple pages in a single browser, each page can have its own viewport size.
 
+#### page.startScreencast(options)
+- `options` <[Object]>
+  - `format` <"png" | "jpeg"> Specify screencast frame format, could be either `jpeg` or `png`. Defaults to 'png'.
+  - `quality` <[number]> The quality of the image, between 0-100. Not applicable to `png` images.
```

Comment:
```suggestion
  - `quality` <[number]> The quality of the image, between 0-100. Not applicable to `'png'` images.
```

Reply:
`png` is consistent with the docs @ `page.screenshot([options])` :-)